samples/boards/reel_board/mesh_badge/src/reel_board.c

```diff
@@ -67,10 +65,14 @@ static struct {
 	struct device *dev;
 	const char *name;
 	u32_t pin;
+	gpio_flags_t flags;
 } leds[] = {
-	{ .name = DT_ALIAS_LED0_GPIOS_CONTROLLER, .pin = DT_ALIAS_LED0_GPIOS_PIN, },
-	{ .name = DT_ALIAS_LED1_GPIOS_CONTROLLER, .pin = DT_ALIAS_LED1_GPIOS_PIN, },
-	{ .name = DT_ALIAS_LED2_GPIOS_CONTROLLER, .pin = DT_ALIAS_LED2_GPIOS_PIN, },
+	{ .name = DT_ALIAS_LED0_GPIOS_CONTROLLER, .pin = DT_ALIAS_LED0_GPIOS_PIN,
```

Comment:
You could just do `DT_ALIAS_LED0_GPIOS` which expands to the controller name, pin, and flags in a single initializer expression.  Though without a pre-defined standard structure synchronized to generation of that expression it's technically more future-proof to spelling out each element.

Reply:
Oh, thanks for letting me know of this DT-generated macro. As you yourself mention, since this uses designated initializers I'd rather not change it at this point.